src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -11,6 +11,8 @@ namespace OpenTelemetry.Metrics;
 
 internal sealed class AggregatorStore
 {
+    // Constant to account for additional space for overflow attribute and a case with no dimensions.
+    internal const int AdditionalReserve = 2;
```

Comment:
Probably no need to have this constant unless it's going to be used in multiple places.